dev-vcs/qgit/qgit-2.9.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="A Qt-based git GUI viewer"
+DESCRIPTION="With qgit you will be able to browse revisions history,\
+view patch content and changed files, graphically following different \
+development branches."
+HOMEPAGE="https://github.com/tibirna/qgit"
+COPYRIGHT="2011-2019 Cristian Tibirna"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/tibirna/qgit/archive/qgit-$portVersion.tar.gz"
+CHECKSUM_SHA256="69788efda7d434d1bc094bb414bd92c269dc7894326320634500b05d63c008e8"
+SOURCE_DIR="qgit-qgit-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	qgit$secondaryArchSuffix = $portVersion
+	cmd:qgit
```

Comment:
Needs $secondaryArchSuffix
```diff
- 	cmd:qgit
+ 	cmd:qgit$secondaryArchSuffix
```

Reply:
So all of the requirements needs $secondaryArchSuffix